### DISCUSSION TITLE
RESUME playlist feature

### DISCUSSION THREAD

**Discussion by Cubicle13** [libre-tube/LibreTube#3535] [Ideas]
I really like the libreTube app. I like it more than a lot of other YouTube alternatives. There's one VITAL FUNCTION that seems to be missing... RESUMING a playlist; ideally this would be a button next to "play all" or automatically when restarting a playlist video from history.

**Comment by Cubicle13** [comment]
Almost forgot to define what that would look like; the feature would EITHER play from the last video played from the list, OR play from the first video that wasn't completely watched yet. The first option has my preference.

**Comment by Silther** [comment]
Dupe of #3330

**Comment by Silther** [comment]
@Cubicle13
Please close this discussion
